templates/fblogin/index.html

```diff
@@ -0,0 +1,10 @@
+<html>
```

Comment:
You have same HTML code in all files. Take a look on template inheritance in Django